crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -433,10 +433,7 @@ impl MemoryStore {
 
     /// Get the user IDs for the given room with the given memberships and
     /// stripped state.
-    ///
-    /// If `memberships` is empty, returns all user IDs in the room with the
-    /// given stripped state.
```

Comment:
Why delete this part of the docs?

Reply:
I think it could be useful. And the constructor for the empty set is named `empty` so it's still clear what this is talking about I think.